tests/Auth/AuthGuardTest.php

```diff
@@ -282,7 +282,7 @@ public function testOnceUsingIdSetsUser()
         $guard->getProvider()->shouldReceive('retrieveById')->once()->with(10)->andReturn($user);
         $guard->shouldReceive('setUser')->once()->with($user);
 
-        $this->assertTrue($guard->onceUsingId(10));
+        $this->assertEquals($user, $guard->onceUsingId(10));
```

Comment:
In fact there are a few other occurrences of `assertEquals($user, ...)` in this file. Not really worth bothering about this, though.